commandline/WSL/install_guide.md

```diff
@@ -1,84 +1,93 @@
+## Prerequisites
+Your PC must be running a 64-bit version of Windows 10.
 
-##Prerequisites
-1. Your PC must be running (at a minimum) a 64-bit version of Windows 10 Anniversary Update. **Creators Update is recommended.**
+To find your PC's architecture and Windows build number, open  
+**Settings** > **System** > **About**
 
-> To find your PC's CPU architecture and Windows version/build number, open 
-> **Settings**>**System**>**About**. 
-> Look for the **OS Build** and **System Type** fields.  
+Look for the **OS Build** and **System Type** fields.  
+    ![](media/system.png) 
 
-![](media/system.png) 
+## Install the Windows Subsystem for Linux
+Enable the "Windows Subsystem for Linux" optional feature and reboot.
 
-If your build is below 14393, try checking for updates.
+1. Open PowerShell as Administrator and run:
+    ``` PowerShell
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+    ```
 
+2. Restart your computer when prompted.
 
-## Installation
-In order to run Bash on Windows, you will need to manually:
-1. Turn-on Developer Mode
-2. Enable the "Windows Subsystem for Linux (beta)" feature via the GUI or the command-line:
+## For Windows Insiders -- Install Linux distribution of choice
 
-### Turn on Developer Mode
-1. Open Settings -> Update and Security -> For developers
-2. Select the Developer Mode radio button  
-  ![](media/updateAndSecurity.png)
+> This section is for Windows Insiders (build 16215 or later).  For earlier versions of Windows 10, follow [these instructions using lxrun](install_guide.md#Install-using-lxrun-Anniversary-Update-and-Creators-Update).
 
-### Enable the Windows Subsystem for Linux feature (GUI)
-1. From Start, search for "Turn Windows features on or off" (type 'turn')
-2. Select **Windows Subsystem for Linux (beta)**  
-  ![](media/windowsFeatures.png)
-  
-3. Click OK
+1. Open the Windows Store and choose your favorite Linux distribution.      
+    Here are links directly to the store installers:
+    * [Ubuntu](https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6)
+    * [OpenSUSE](https://www.microsoft.com/store/apps/9njvjts82tjx)
+    * [SLES](https://www.microsoft.com/store/apps/9p32mwbh6cns)
 
-### Enable the Windows Subsystem for Linux feature (command-line)
+    ![](media/UbuntuStore.png)
 
-Open a PowerShell prompt as administrator and run:
+2. Select "Get"
 
-```PowerShell
-Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
-```
+    >  **Troubleshooting:  Installation failed with error 0x8007007e**  
+    >  This error occurs when your system doesn't support Linux from the store.  Make sure that:
+    > * You're running Windows build 16215 or later. [Check your build](install_guide.md#prerequisites).
+    > * The Windows Subsystem for Linux optional component is enabled. [Instructions here](install_guide.md#Install-the-Windows-Subsystem-for-Linux).

```

Comment:
Can we mention that installing on non-system drive is not supported and they might see the error code 0x80070003